core/templates/pages/exploration-editor-page/modal-templates/metadata-version-history-modal.component.ts

```diff
@@ -78,7 +78,7 @@ export class MetadataVersionHistoryModalComponent
     return this.versionHistoryService.canShowForwardMetadataDiffData();
   }
 
-  getLastEditedVersionNumber(): number {
+  getLastEditedVersionNumber(): number | null {
```

Comment:
Add a docstring and explain what a return value of null would represent.

Reply:
done